File: Difficulty 1.0-1.9/ratingproblems.java
import java.io.*;
import java.util.*;
public class ratingproblems {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());

        int n = Integer.parseInt(st.nextToken());
        int k = Integer.parseInt(st.nextToken());
        double sum = 0;

        for(int i = 0; i < k; i++) {
            sum += Integer.parseInt(br.readLine());
        }

        double min = (sum + ((n - k) * -3)) / n; 
        double max = (sum + ((n - k) * 3)) / n;

        System.out.println(min + " " + max);
    }
}

File: C++/ratingproblems.cpp
#include <iostream>
using namespace std;

int main() {
  double k, r;
  cin >> k >> r;

  double min = 0;
  double max = 0;

  for (int i = 0; i < r; i++) {
    double j;
    cin >> j;
    min += j;
    max += j;
  }

  min += ( (k - r) * -3);
  max += ( (k - r) * 3);

  cout << min/k << " " << max/k << endl;
}